examples/encrypted_filesystem/Dockerfile.primary

```diff
@@ -1,4 +1,6 @@
-FROM python:latest
+FROM mcr.microsoft.com/cbl-mariner/base/python:3.9
+
 WORKDIR /app
 COPY encrypted_filesystem/payload.py payload.py
+RUN alias python=python3
```

Comment:
why not just update the command to "python3"?

Reply:
fair, I'll change it